index.test.js

```diff
@@ -0,0 +1,36 @@
+const { createEvent } = require('@posthog/plugin-scaffold/test/utils')
+const { processEvent } = require('.')
+
+const global = { propertiesToFilter: ['gender', '$set.age', 'foo.bar.baz.one', 'nonExisting'] }
```

Comment:
@yakkomajuri I guess using `.` here to define nesting is a reasonably safe separator? ie. posthog itself doesn't use `.` in property names anywhere?